src/main/java/uk/gov/hmcts/divorce/common/notification/Applicant1AppliedForFinalOrderNotification.java

```diff
@@ -164,4 +193,36 @@ private Map<String, String> applicant1TemplateVars(CaseData caseData, Long id) {
 
         return templateVars;
     }
+
+    private void generateAndSendFinalOrderAppliedForToSolicitor(CaseData caseData, Long caseId, Applicant applicant) {
+
+        Document generatedDocument = generateDocument(caseId, applicant, caseData);
+
+        Letter letter = new  Letter(generatedDocument, 1);
+        String caseIdString = String.valueOf(caseId);
+
+        final Print print = new Print(
+                List.of(letter),
+                caseIdString,
+                caseIdString,
+                FINAL_ORDER_REQUESTED,
+                applicant.getFullName(),
+                applicant.getAddressOverseas()
```

Comment:
Should we be getting the sol details here?

Reply:
I believe you are correct. I have changed it to solicitor overseas